packages/tpetra/core/test/ImportExport2/ImportExport2_UnitTests.cpp

```diff
@@ -2845,11 +2845,20 @@ TEUCHOS_UNIT_TEST_TEMPLATE_2_DECL( Import_Util,GetTwoTransferOwnershipVector, LO
   TEUCHOS_UNIT_TEST_TEMPLATE_2_INSTANT( Import_Util, GetPids, LO, GO ) \
   TEUCHOS_UNIT_TEST_TEMPLATE_2_INSTANT( Import_Util, GetTwoTransferOwnershipVector, LO, GO )
 
-#define UNIT_TEST_GROUP_SC_LO_GO( SC, LO, GO )                   \
+#define UNIT_TEST_GROUP_SC_LO_GO_COMMON( SC, LO, GO )                   \
   TEUCHOS_UNIT_TEST_TEMPLATE_3_INSTANT( CrsMatrixImportExport, doImport, LO, GO, SC ) \
   TEUCHOS_UNIT_TEST_TEMPLATE_3_INSTANT( FusedImportExport, doImport, LO, GO, SC ) \
-  TEUCHOS_UNIT_TEST_TEMPLATE_3_INSTANT( Import_Util, UnpackAndCombineWithOwningPIDs, LO, GO, SC ) \
-  TEUCHOS_UNIT_TEST_TEMPLATE_3_INSTANT( FusedImportExport, MueLuStyle, LO, GO, SC )
+  TEUCHOS_UNIT_TEST_TEMPLATE_3_INSTANT( Import_Util, UnpackAndCombineWithOwningPIDs, LO, GO, SC )
+
+// FIXME_SYCL requires quering free device memory
```

Comment:
Please add better comments here.  What is this code doing, and what needs to be fixed?  Why is the _COMMON macro needed?  Does the MueLuStyle test not work with SYCL, and if so, why not?   Better details in the code for FIXME issues will help the future fixers know what needs to be done.  Thanks.

Reply:
`FusedImportExport` tries to query for free device memory somewhere in Kokkos Kernels and this is not supported in the SYCL specifications.
To preserve the definition of `UNIT_TEST_GROUP_SC_LO_GO`, I introduced `UNIT_TEST_GROUP_SC_LO_GO_COMMON` to only disable the test mentioned above for `SYCL`.